jmh.gradle

```diff
@@ -33,7 +33,7 @@ if (sparkVersions.contains("3.0")) {
 }
 
 if (sparkVersions.contains("3.2")) {
-  jmhProjects.add(project(":iceberg-spark:iceberg-spark-3.2"))
+  jmhProjects.add(project(":iceberg-spark:iceberg-spark-3.2_2.12"))
```

Comment:
Actually, I'm not sure that we should. Right now, we don't actually support other versions and if we use a plugin it might have a different way to do this. Let's leave it as-is for now.

Reply:
Oh here I mean that it's presently possible to run JMH tests on Spark 3.0 and Spark 3.2, but not Spark 3.1.

Adding Spark 3.1 support to the JMH tests feels orthogonal to adding Scala versioning.

Reply:
Sorry, I meant to put this on my other comment. I agree with you on the JMH follow-up.

Reply:
Ah for the Scala version in the matrix parameters you mean? I can revert that back to hard-coded `_2.12` if we prefer.

Reply:
Moved back to hardcoded `_2.12` in the GithubActions file for spark-ci.

We can sort it out once we land on a plugin or method of supporting several Scala versions.